pkg/apis/networking/v1/defaults.go

```diff
@@ -43,3 +45,13 @@ func SetDefaults_NetworkPolicy(obj *networkingv1.NetworkPolicy) {
 		}
 	}
 }
+
+func SetDefaults_IngressClass(obj *networkingv1.IngressClass) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.IngressClassNamespacedParams) {
+		return
+	}
+	if obj.Spec.Parameters == nil {
+		return
+	}
+	obj.Spec.Parameters.Scope = networkingv1.IngressClassParametersReferenceScopeCluster
```

Comment:
Still need to ensure that `Scope` is not set before you set a default value here.